src/license_guc.h

```diff
@@ -13,61 +13,22 @@
 #include <export.h>
 #include <guc.h>
 
-/*
- * The license for using only Apache features is 'ApacheOnly'
- *
- * For documentation on other license types,
- * and information on the license format,
- * see tsl/src/license.c
- * (NB: This requires accepting LICENSE-TIMESCALE)
- */
-typedef enum LicenseType
-{
-	LICENSE_TYPE_APACHE_ONLY = 'A',
-	LICENSE_TYPE_COMMUNITY = 'C',
-	LICENSE_TYPE_ENTERPRISE = 'E',
-} LicenseType;
-
-#define TS_APACHE_ONLY_LICENSE "ApacheOnly"
-#define TS_COMMUNITY_LICENSE "CommunityLicense"
+#define TS_LICENSE_APACHE "apache"
+#define TS_LICENSE_TSL "tsl"
```

Comment:
why not use apache and community here that's we call them externally?

Reply:
I am fine with community. I think we just need to discuss and decide which name we would like to use